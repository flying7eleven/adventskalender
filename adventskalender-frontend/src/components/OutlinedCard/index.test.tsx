import { create } from 'react-test-renderer';
import { OutlinedCard } from './index';

describe('OutlinedCard component', () => {
    test('Matches the snapshot', () => {
        const card = create(<OutlinedCard headline={'Headline'} value={'Value'} description={'Description'} />);
        expect(card.toJSON()).toMatchSnapshot();
    });
});
