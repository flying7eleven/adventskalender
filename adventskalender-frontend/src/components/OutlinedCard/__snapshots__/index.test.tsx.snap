// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OutlinedCard component Matches the snapshot 1`] = `
<div
  className="MuiBox-root css-12dv9jb"
>
  <div
    className="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root css-196bwzp-MuiPaper-root-MuiCard-root"
    style={{}}
  >
     
    <div
      className="MuiCardContent-root css-1lt5qva-MuiCardContent-root"
    >
      <p
        className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-qeip0x-MuiTypography-root"
        style={{}}
      >
        Headline
      </p>
      <div
        className="MuiTypography-root MuiTypography-h3 css-13msox4-MuiTypography-root"
        style={{}}
      >
        Value
      </div>
      <p
        className="MuiTypography-root MuiTypography-body1 css-nsr5mc-MuiTypography-root"
        style={{}}
      >
        Description
      </p>
    </div>
  </div>
</div>
`;
